packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -165,6 +166,11 @@ export abstract class LibraryCommandletExecutor<T>
       channelService.showCommandWithTimestamp(
         `${nls.localize('channel_end')} ${this.executionName}`
       );
+
+      if (this.showChannelOutput) {
```

Comment:
Right now is showChannelOutput always going to be true?

Reply:
Yes ma'am it's defaulting to true for the library based commands